infra/terraform/base_stacks_2/README.md

```diff
@@ -0,0 +1,40 @@
+# Before applying changes
+
+For initial configuration please refer to the README file located in ./infra/terraform
+
+This base_stacks_2 directory is only to be deployed after the deployment of the original base_stacks as dns_records.tf is dependent on dns.tf being deployed first.
+
+# Applying changes
```

Comment:
Please remove these instructions as it is duplicating the details from the `./terraform/README.md` (née `USAGE.md`) document. The aim is to keep both the `base-stacks` and `base-stacks-2` terraform configurations structured in the same way so we should be able to cover both with the same document. 

---- 

My preference would be to keep the `USAGE.md` document as a concise reference for the commands, with a `README.md` to detail out more the reasoning for each tf-config, but I'm also happy to follow your lead with any improvements to the documentation.

Reply:
Yep makes sense!

I only renamed the `USAGE.md` to `README` as its a mono repo and the use of `README` already seems established, happy to revert it back to `USAGE` if we also potentially end up adding a `README`.

Reply:
As long as the document name and its contents / intention line up then we're in a good place.